server/src/app.ts

```diff
@@ -22,5 +23,13 @@ const io = socketio(server);
 //     callback();
 //   });
 // });
+
+const dbKey = process.env.DB_KEY;
+
+mongoose
+  .connect(dbKey, { useNewUrlParser: true, useUnifiedTopology: true })
+  .then(() => console.log("Connected with DB"))
+  .catch(() => console.error("Error with DB"));
```

Comment:
Od niedawna mamy w TS możliwość wykorzystania await w globalnym zasięgu bez funkcji asynchronicznej. Może spróbujemy to zamienić na try catcha z await? Linii kodu wyjdzie pewnie tyle samo ale wydaje mi się że będzie ładniej. Zostawiam tobie do decyzji.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html